### ISSUE TITLE
Allow autoFocus

### ISSUE THREAD

**Issue by steida** [JedWatson/react-select#400]
Should render autoFocus on input.

``` jsx
<Select
  autoFocus
  onChange={actions.search.onAddressAutocompleteChange}
  placeholder={msg.placeholder}
  value={search.addressAutocompleteValue}
/>
```

**Comment by wbinnssmith** [comment]
As it turns out this *is* actually supported, but the prop is `autofocus`, unlike the JSX/DOM api for regular dom inputs. @JedWatson what do you think of using `autoFocus` instead for consistency with those apis? Probably ought to support both in the meantime as to not break the current api.

**Comment by 1Jesper1** [comment]
Why is the prop autofocus depricated then?

**Comment by JedWatson** [comment]
I didn't link this issue when I fixed this (there were a couple) - as of `v1.0.0` the prop was renamed to `autoFocus` and the old `autofocus` prop has been deprecated but continues to work in 1.x for backwards compatibilty.

The `autofocus` prop will be removed in `v2.0.0`
